src/shifter.c

```diff
@@ -551,7 +551,7 @@ static void _usage(int status) {
     printf("\n"
         "Usage:\n"
         "shifter [-h|--help] [-v|--verbose] [--image=<imageType>:<imageTag>]\n"
-        "    [--entry] [-V|--volume=/path/to/bind:/mnt/in/image[:<flags>][,...]]\n"
+        "    [--entry] [-V|--volume=/path/to/bind:/mnt/in/image[:<flags>][,...][;...]]\n"
```

Comment:
while this is all confusing:
this is the needed string:
[-V|--volume=/path/to/bind:/mnt/in/image[:<flags>[,...]][;...]]

note that the comma flag is bracketed within the flags bracket, whereas the semicolon is related to the volume option